# TiDeTree: A Bayesian phylogenetic framework to estimate single-cell trees and population dynamic parameters from genetic lineage tracing data

Sophie Seidel<sup>1,2</sup>, Tanja Stadler<sup>1,2</sup>

<sup>1</sup>ETH Zurich, Department of Biosystems Science and Engineering, 4058 Basel, Switzerland

<sup>2</sup>Swiss Institute of Bioinformatics (SIB), Switzerland


Analyses and figures associated with the [TiDeTree manuscript](https://doi.org/10.1101/2022.02.14.480422)

## Abstract
The development of a multicellular organism is governed by an elaborate balance between cell division, death, and differentiation. These core developmental processes can be quantified from single-cell phylogenies. Here we present TiDeTree, a Bayesian phylogenetic framework for inference of time-scaled single-cell phylogenies and population dynamic parameters such as cell division, death, and differentiation rates from genetic lineage tracing data. We show that the performance of TiDeTree can be improved by incorporating multiple sources of additional independent information into the inference. Finally, we apply TiDeTree to a lineage tracing dataset to estimate time-scaled phylogenies, cell division, and apoptosis rates. We envision TiDeTree to find wide application in single-cell lineage tracing data analysis which will improve our understanding of cellular processes during development. The source code of TiDeTree is publicly available at https://github.com/seidels/tidetree.

